src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -710,15 +738,16 @@ export class PoolFormComponent extends CdForm implements OnInit {
       return;
     }
     if (usage) {
-      deletionBtn.animation = false;
-      deletionBtn.toggle();
-      this.data[dataName] = true;
-      setTimeout(() => {
-        const tabs = getTabs();
-        if (tabs) {
-          tabs.select(tabPosition);
-        }
-      }, 50);
+      const isOpen = this.data[dataName];
```

Comment:
```suggestion
      const isOpen = this.data?.[dataName];
```
or make sure data is always defined